### ISSUE TITLE
Loading the converted Keras model takes a long time

### ISSUE THREAD

**Issue by miranthajayatilake** [tensorflow/tfjs#492]
Hi,

I'm loading a model via the 'model.json' URL and it appears to take a long time to process. My code is as below:

```
const MODEL_PATH = 'https://raw.githubusercontent.com/miranthajayatilake/pix2pix/master/model.json';
pixmodel = await tf.loadModel(MODEL_PATH)

```
I have uploaded the json file and the group files to GitHub, and used the link to the raw file, as I found this method convenient. 

The size of the files add up to about 170MB, where I think this can be a problem. But I have used other models such as mobilenet where they load very fast. 

So is my model taking a long time to load because of the size or is it due some other reason? If it's the size, are there any measures to reduce the size of the files?

Also are there any other ways to provide this json file via a locally hosted server? (so it doesn't depend on the internet speed)

Thanks!

**Comment by miranthajayatilake** [comment]
Thanks for the suggestions @caisq ! I recorded the network download status. See the attachments below;

![screenshot from 2018-07-06 08-46-10](https://user-images.githubusercontent.com/11708057/42358692-8ff5f2f8-80fb-11e8-9011-8c7f872d11c2.png)

As seen here, some of the group files takes more than 4 mins to download. This seems to be the bottleneck. The total time and data transfer was as below;

![screenshot from 2018-07-06 08-47-08](https://user-images.githubusercontent.com/11708057/42358744-d9bdfaac-80fb-11e8-80fa-3f993b5854ff.png)

I guess it is clear that the bulkiness of the model is affecting the time taken. As suggested I tested with loading the smaller 'mobilenet' model. The time taken was very less.

![screenshot from 2018-07-06 08-59-44](https://user-images.githubusercontent.com/11708057/42358780-06e19f02-80fc-11e8-9d1f-765f7a41c5f9.png)

So I suppose I'll have to look into ways of reducing the model size right? 
Just out of curiosity, are there any means of hosting the model locally and loading it? That would be good to know. 

Nevertheless, in order to reduce the size of the model, as I understand the option would be to change the architecture of the model. I would like to hear any suggestions on this as well.

Thanks a lot!
